.github/workflows/connect-transport-e2e-test.yml

```diff
@@ -0,0 +1,48 @@
+name: Transport E2E Tests
```

Comment:
You should adhere to the naming convention.

Reply:
:+1: https://github.com/trezor/trezor-suite/pull/11609/commits/b52b5e0dd372879acf671166504ef658cfc20cbd